_data/hosts.yml

```diff
@@ -70,13 +70,9 @@
     name: Aruba.it
     url: 'http://hosting.aruba.it/'
     type: shared
-    default: 5.5.28
+    php70: 0
```

Comment:
Thanks for the note. 

PHP 7.0 is not offered in Aruba right now, going to push a fix by removing the offending line.